### PR TITLE
modified how to send files

### PR DESCRIPTION
**Pull request by jalogisch**
after modified https://github.com/Graylog2/graylog-guide-windows-eventlog/commit/8a7d1715688481b73b0426add7a2576242eb5089 i notice that this has also some older text in it and updated it to the present technical level

### CODE REVIEW

File: pages/sending_data.rst [hunk 1]
```diff
@@ -249,8 +249,6 @@ You can `learn more about JSONPath here <http://goessner.net/articles/JsonPath/>
 Reading from files
 ==================
 
-Graylog is currently not providing an out-of-the-box way to read log messages from files. We do however recommend two
-fantastic tools to do that job for you. Both come with native Graylog (GELF) outputs:
+Graylog itself is not able to read log messages from file, but it comes with the :ref:`Collector Sidecar <graylog-collector-sidecar>` which can used as a supervisor for tools that are build for this purpose.
```
Comment (joschi) [inline pages/sending_data.rst]: Maybe rephrase that altogether because with a plugin providing a file input, Graylog would be able to read files (from the local system). It's just not possible out-of-the-box.

Also missing word: "which can _be_ used as"

File: pages/sending_data.rst [hunk 2]
```diff
@@ -249,8 +249,6 @@ You can `learn more about JSONPath here <http://goessner.net/articles/JsonPath/>
...
 
-  * `fluentd <http://www.fluentd.org/guides/recipes/graylog2>`__
-  * `logstash <http://logstash.net/docs/1.4.2/outputs/gelf>`__
+Additional any tool that can read files and send native Graylog (GELF) messages can be used.
```
Comment (joschi) [inline pages/sending_data.rst]: …or plaintext (e. g. netcat) or syslog or basically anything for which an input exists.

File: pages/sending_data.rst [hunk 3]
```diff
@@ -249,7 +249,6 @@ You can `learn more about JSONPath here <http://goessner.net/articles/JsonPath/>
 Reading from files
 ==================
 
-Graylog itself is not able, out-of-the-box to read log messages from file. But we had build the :ref:`Collector Sidecar <graylog-collector-sidecar>` which can be used as a supervisor for tools that are made for this purpose.
-
-Any tool that can send data to Graylog can be used, as long as an input is present in Graylog that is able to receive the data.
+Graylog ifself is not able to read log messages from file. We had build the :ref:`Collector Sidecar <graylog-collector-sidecar>` that can act like a supervisor for tools that made for this purpose.
 
+This leaves the ability untouched that you can use any tool that can create a valid message for the present Graylog inputs.
```
Comment (joschi) [inline pages/sending_data.rst]: I don't understand that sentence.

Comment (jalogisch) [inline pages/sending_data.rst]: my idea was to honor your comment: https://github.com/Graylog2/documentation/pull/148#discussion_r73474153

and say that anything that can create a message that can understood by any input that graylog has.
